server/app/rest_api/rest.py

```diff
@@ -201,8 +217,8 @@ def get(self):
         try:
             annotation_response = current_app.data.annotation(current_app.data.data, "obs", fields)
         except KeyError:
-            return make_response(f"Error bad key in {fields}", 404)
-        return make_response(jsonify(annotation_response))
+            return make_response(f"Error bad key in {fields}", HTTPStatus.NOT_FOUND)
```

Comment:
I think this should probably be `BAD_REQUEST`

Reply:
@bkmartinjr NOT_FOUND is in the spec, but I'm with @ttung on this one. NOT_FOUND has the implication that the resource doesn't exist, not just that a user supplied bad values.

Reply:
Makes sense to me.  Please change the spec as well as the code.

Reply:
will do